input/pagecontent/overview.md

```diff
@@ -252,33 +252,33 @@ In this section, we will walk through a realistic example of FSH, line by line.
 ```
 
 * Lines 1 and 2 define aliases for the LOINC and SNOMED-CT code systems.
-* Line 4 declares the intent to create a profile with the name CancerDiseaseStatus. The name is typically PascalCase (also known as UpperCamelCase) and according to FHIR, should be "[usable by machine processing applications such as code generation](https://hl7.org/fhir/structuredefinition.html#resource)".
+* Line 4 declares the intent to create a profile with the name `CancerDiseaseStatus`. The name is typically PascalCase (also known as UpperCamelCase) and according to FHIR, should be "[usable by machine processing applications such as code generation](https://hl7.org/fhir/structuredefinition.html#resource)".
 * Line 5 says that this profile will be based on Observation.
 * Line 6 gives an id for this profile. The id is used to create the globally unique URL for the profile. The URL is composed of the IG’s canonical URL, the instance type (always `StructureDefinition` for profiles), and the profile’s id.
 * Line 7 is a human-readable title for the profile.
 * Line 8 is the description that will appear in the IG on the profile's page.
-* Line 9 is the start of the rule section of the profile. It uses [caret syntax](reference.html#caret-paths) to set the status attribute in the StructureDefinition produced for this profile.
+* Line 9 is the start of the rule section of the profile. It uses [caret syntax](reference.html#caret-paths) to set the `status` attribute in the StructureDefinition produced for this profile.
 * Line 10 adds an extension to the profile using the standalone extension, `EvidenceType`, gives it the local name `evidenceType`, and assigns the cardinality 0..*. _EvidenceType is defined on line 31._
-* Line 11 binds the valueCodeableConcept of the evidenceType extension to a value set named CancerDiseaseStatusEvidenceTypeVS with a required binding strength. _CancerDiseaseStatusEvidenceTypeVS is defined on line 47._
+* Line 11 binds the `valueCodeableConcept` of the evidenceType extension to a value set named `CancerDiseaseStatusEvidenceTypeVS` with a required binding strength. _CancerDiseaseStatusEvidenceTypeVS is defined on line 47._
```

Comment:
There are a few places where I've decided to forgo one styling when multiple styles apply. For example, when a type name is linked to something, I do not apply the code styling because it eliminates the blue font color and underline (and no longer looks like a link).

In this case, I feel like the italicized statement is more important than the fact that CancerDiseaseStatusEvidenceTypeVS is a type. I assumed that the code styling would take precedence and the name would not be italicized at all. But it turns out it italicizes it _and_ applies a background shade:
![CleanShot 2024-03-05 at 18 23 37@2x](https://github.com/HL7/fhir-shorthand/assets/2278253/80e0c133-1235-4025-9149-0f33364f9158)

I'm not sure how I feel about that. In my head, code styling should never be italicized. I feel like non-italic is part of the code style. So when I see it italicized, it doesn't _feel_ like code styling to me. That said, the markdown renderer used by IGP obviously has a different opinion on that. What do you think?

Reply:
After discussion, we will leave this as-is, because code styling + fixed width text + italic styling is "just too much."